packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -133,11 +118,7 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
   );
 
   const onRenderChildren = (): JSX.Element => {
-    return (
-      <Stack horizontal className={mergedChildrenStyle}>
-        {children}
-      </Stack>
-    );
+    return <>{children}</>;
```

Comment:
Do we need anything specific to ensure the children are contained with the `mergeChildrenStyle`?

Reply:
no, the styles used in `mergeChildrenStyle` are partially removed (because they aren't needed anymore), partially migrated to the children styles